code/datums/tutorial/marine/role_specific/hospital_corpsman_intermediate.dm

```diff
@@ -0,0 +1,1479 @@
+	var/stage = 0
+	var/cpr_count = 0
+	/// For use in the handle_pill_bottle helper, should always be set to 0 when not in use
+	var/handle_pill_bottle_status = 0
+
+	var/list/vendor_failsafe = list()
+
+	var/obj/limb/chest/mob_chest
+
+// ------------ CONTENTS ------------ //
+//
+// Section 1 - Stabilizing Types of Organ Damage
+// 1.1 Internal Organ Damage (Chest)
+// 1.2 Heart Damage
+// 1.3 Lung Damage
+// 1.4 Internal Organ Damage (Head)
+// 1.5 Liver and Kidney Damage
+//
+// Section 2 - Revivals
+// 2.1 Defibrillations
+// 2.2 Revival Conditions
+// 2.3 Delivering CPR
+// 2.4 Assisted Revivals
+//
+// Section 3 - Field Surgery
+// 3.1 Surgical Damage Treatment
+// 3.2 Internal Bleeding
+//
+// Section 4 - Medical Evacuations, Stasis
+//
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_intermediate/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	init_mob()
+	message_to_player("This tutorial will teach you the more complex elements of playing as a Marine Hospital Corpsman.")
+	addtimer(CALLBACK(src, PROC_REF(uniform)), 4 SECONDS)
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_intermediate/proc/uniform()
+	SIGNAL_HANDLER
+
+	message_to_player("Before you're ready to take on the world as a Marine Hospital Corpsman, you should probably put some clothes on...")
+	message_to_player("Stroll on over to the outlined vendor and vend everything inside.")
+	update_objective("Vend everything inside the ColMarTech Automated Closet.")
+
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/structure/machinery/cm_vending/clothing/tutorial/medic/advanced, medical_vendor)
+	add_highlight(medical_vendor, COLOR_GREEN)
+	medical_vendor.req_access = list()
+	RegisterSignal(medical_vendor, COMSIG_VENDOR_SUCCESSFUL_VEND, PROC_REF(uniform_vend))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_intermediate/proc/uniform_vend(datum/source, obj/structure/machinery/cm_vending/vendor, obj/item/new_item)
+	SIGNAL_HANDLER
+
+	clothing_items_to_vend--
+	vendor_failsafe |= new_item
+	if(clothing_items_to_vend <= 0)
+		TUTORIAL_ATOM_FROM_TRACKING(/obj/structure/machinery/cm_vending/clothing/tutorial/medic/advanced, medical_vendor)
+		UnregisterSignal(medical_vendor, COMSIG_VENDOR_SUCCESSFUL_VEND)
+		medical_vendor.req_access = list(ACCESS_TUTORIAL_LOCKED)
+		remove_highlight(medical_vendor)
+
+		var/obj/item/storage/belt/medical/lifesaver/medbelt = locate(/obj/item/storage/belt/medical/lifesaver) in vendor_failsafe
+		add_to_tracking_atoms(medbelt)
+		var/obj/item/device/healthanalyzer/healthanalyzer = new(loc_from_corner(0, 4))
+		add_to_tracking_atoms(healthanalyzer)
+		add_highlight(healthanalyzer, COLOR_GREEN)
+		message_to_player("Great. Now pick up your trusty <b>Health Analyzer</b>, and let's get started with the tutorial!")
+		update_objective("")
+		RegisterSignal(tutorial_mob, COMSIG_MOB_PICKUP_ITEM, PROC_REF(organ_tutorial))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_intermediate/proc/organ_tutorial()
```

Comment:
From here to the stethoscope, there's a lot of text being vomited at the player without any interaction.